### ISSUE TITLE
AdGuard DNS breaks Amazon Alexa

### ISSUE THREAD

**Issue by akmLaVx** [AdguardTeam/AdGuardHome#232]
### Steps to reproduce
1. Not provided
2.
3.
4.

### Expected behavior
Shouldn't interfere with Amazon Alexa

### Actual behavior
Amazon Alexa is fully broken

### Your environment

|   Description  |     Value    |
| -------------- | ------------ |
| DNS server in use:| Default
| How did you setup DNS configuration:| Router
| Device model:| PC
| Operating system and version:| Windows 8.1

**Comment by vozersky** [comment]
need more details

**Comment by vozersky** [comment]
what device is customer using exactly? Echo?

@akmLaVx

**Comment by dzamyshlyayeva** [comment]
here is a reply from the customer: "Amazon Echo Dot G2. She just say Echo Dot lost its connection"

**Comment by LunarstarPony** [comment]
Oh Hai there :D

**Comment by ameshkov** [comment]
@vozersky what about our test Echo? Is it working?

**Comment by vozersky** [comment]
@ameshkov  yep, it works, can't reproduce

**Comment by LunarstarPony** [comment]
So it's my Echo Dot Problem? Or

**Comment by vozersky** [comment]
@LunarstarPony  Not sure, how often does it happen? Does it disconnect from wi-fi network completely?

**Comment by LunarstarPony** [comment]
Nope not completely

**Comment by LunarstarPony** [comment]
Only a few of server I think

**Comment by ameshkov** [comment]
What we need to know is how exactly is it broken and if there's any way for us to reproduce this issue

**Comment by ameshkov** [comment]
How often does it happen? What do you need to do to restore it? Is it enough to restart Echo?

**Comment by LunarstarPony** [comment]
Everytime

**Comment by LunarstarPony** [comment]
Just close ADGuard DNS And it back to normal?

**Comment by ameshkov** [comment]
Can you access the router DNS requests log? That'd help a lot if we understood what exact requests are done by Echo

**Comment by LunarstarPony** [comment]
My Router doesn't seem have DNS Log?

**Comment by ameshkov** [comment]
What exact router do you use?

**Comment by LunarstarPony** [comment]
Edimax AC1200

**Comment by ameshkov** [comment]
Bad-bad, I can't see how log DNS requests there. I guess we need to wait for a self-hosted AG DNS solution that'll provide such functionality :(

**Comment by LunarstarPony** [comment]
TAT

**Comment by LunarstarPony** [comment]
Maybe I'll use desktop and Android ADGuard First

**Comment by ameshkov** [comment]
AG for Android won't break it for sure, the problem with these local issues is that we cannot troubleshoot it from here, we just can't see what's going on:(

**Comment by TPS** [comment]
@LunarstarPony Are you able to setup AGHome to finally track this down?  Or has it resolved in the intervening (7️⃣❗) months?

**Comment by godly-devotion** [comment]
Please let me know if you want me to open a separate issue.
I recently switched to using AdGuard DNS on my Asus RT-AC68U and found that I am also having issues with Amazon Alexa as well. Alexa stalls then fails to reply to any queries. I may be able to provide more information since Asuswrt-Merlin is pretty good with logs.

**Comment by LunarstarPony** [comment]
> Please let me know if you want me to open a separate issue.
> I recently switched to using AdGuard DNS on my Asus RT-AC68U and found that I am also having issues with Amazon Echo Dot as well. Alexa stalls then fails to reply to any queries. I may be able to provide more information since Asuswrt-Merlin is pretty good with logs.

It's working on my side rn tho

**Comment by godly-devotion** [comment]
I tried using CloudFlare's DNS with DNSSEC and DoT and it seems to work. I switched back to Adguard's DNS and it is still working now too. I did reboot the device after each DNS so it could either be a brief outage on Amazon's side or Alexa is caching the DNS (which I don't think so since I rebooted it). Lets see, sorry if it ends up not being Adguard's issue.

**Comment by ManishHira-Code** [comment]
In the last 3-4 weeks, all my amazon alexas, refused to respond. Saying they were having trouble connecting. I use both AdGuard (Primary DNS) and PiHole (Secondary DNS) - They both use the same upstream DNS server provider IPs (not google or cloudflare). 

Only after disabling Adguard and rebooting Alexa's (with PiHole now providing DNS), did they start working. My alexas worked for 1 weeks until today, when I turned on Adguard again. Now all of my Alexa's are again in the having trouble connecting .. no other devices are affected .

My Alexa's had no issues when Adguard was primary before the issue surfaced about 3-4 weeks ago so what changed? It's definitely not my upstream DNS provider.

**Comment by iceman73** [comment]
What lists do you have enabled.I had a similar issue with one of my lists and I whitelisted these entries and Alexa and Firesticks came back to life.

aax-us-east.amazon-adsystem.com
device-metrics-us-2.amazon.com
device-metrics-us.amazon.com
cognito-identity.us-east-1.amazonaws.com
unagi-na.amazon.com
ocsp.rootg2.amazontrust.com
det-ta-g7g.amazon.com
todo-ta-g7g.amazon.com
device-messaging-na.amazon.com
s3-eu-west-1.amazonaws.com
us-east-1.amazonaws.com

**Comment by ManishHira-Code** [comment]
I left it as default. Will whitelist these and see over weekend. Thx

________________________________
